### PR TITLE
PhotoElectric for flex

### PR DESCRIPTION
**Pull request by jmunozv**
This PR adds the PhotoElectric process to NextFlex geometry.
Minor improvements in FakeGrid optical material properties and their use from NEW.

### CODE REVIEW

File: source/materials/OpticalMaterialProperties.cc [hunk 1]
```diff
@@ -619,64 +619,36 @@ G4MaterialPropertiesTable* OpticalMaterialProperties::FakeGrid(G4double pressure
...
-    // G4cout << "* FakeGrid rIndex:  " << std::setw(7)
-    //        << ri_energy[i]/eV << " eV -> " << rIndex[i] << G4endl;
-  }
-  assert(sizeof(rIndex) == sizeof(ri_energy));
-  mpt->AddProperty("RINDEX", ri_energy, rIndex, ri_entries);
+  mpt->AddConstProperty("SCINTILLATIONYIELD", xenon_pt->GetConstProperty("SCINTILLATIONYIELD"));
+  mpt->AddConstProperty("RESOLUTIONSCALE",    xenon_pt->GetConstProperty("RESOLUTIONSCALE"));
+  mpt->AddConstProperty("FASTTIMECONSTANT",   xenon_pt->GetConstProperty("FASTTIMECONSTANT"));
+  mpt->AddConstProperty("SLOWTIMECONSTANT",   xenon_pt->GetConstProperty("SLOWTIMECONSTANT"));
+  //mpt->AddConstProperty("ELTIMECONSTANT",     xenon_pt->GetConstProperty("ELTIMECONSTANT"));
```
Comment (gonzaponte) [inline source/materials/OpticalMaterialProperties.cc]: Any reason to keep this commented line?

**Review by gonzaponte — COMMENTED** [review]
Looks good. Just a minor point about a commented line, but this is essentially ready to be approved and merged.

File: source/materials/OpticalMaterialProperties.cc (hunk 1 above)
Comment (jmunozv) [reply source/materials/OpticalMaterialProperties.cc]: It is there for consistency reasons with the GXe but, as this property is not used for any of the materials, I proceed to remove these comments.

**Review by gonzaponte — APPROVED** [review]
Ready to go.

**Review by paolafer — APPROVED** [review]
Approval ratification.

**Pull request merged**